README.md

```diff
@@ -50,6 +51,16 @@ Completing the cloud setup dynamically creates an edge client for your account i
 
 The client exchanges the bootstrap certificates for the production certificates with extended permissions. These certificates are then used for all future mqtt communication on topics containing the unique name assigned to this device (described in detail in fleet-provisioning components).
 
+### NEW! Cert Rotation 
+All things created in the AWS IoT registry will contain a *cert_issuance* attribute which will be the trigger for detecting certificates outside of the expiry date. A lambda trigger will run daily to detect expired certs and publish a message to each endpoint within the graceperiod of expiry. Your edge device must subscribe to this message and handle the published alert (arriving each day until compliance is met or certificate expires altogether). By executing the provided *run_provisioning* method (in main.py) and passing in run_provisioning(isRotation=True), the edge will perform a new cert rotation. The provisioning hook will validate if the cert_issuance date is indeed out of compliance and if so, issue a new certificate with an updated date (deactivating the old). The following illustrates a sample test workflow:
+
+1) Make sure to enable Fleet Index Settings (Thing Indexing) in IoT Core / Settings.
```

Comment:
Can we make this change from CF?

Reply:
We can and it's not hard. I can make this a TODO but won't have time with recent client work for a week or two. Can I ask for a merge with the updated functionality to reconcile my workbench and clear my current train of thought ? I will ensure to come back and roll in the auto-fleet indexing.